### PR TITLE
Lint Mozilla Annual Report JS files

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Lint JS files for annual report pages. This is mainly indentation and single quote changes.

- /foundation/annualreport/2011/
- /foundation/annualreport/2012/
- /foundation/annualreport/2013/

Note: There is some cleanup we could do with regard to DOM querying & caching selectors on the older 2011 page, but that's not the aim right now.

## Bugzilla
N/A

## Testing
Should be a fairly safe change, mostly style related as opposed to changing any logic. Just a quick test to make sure there are no obvious regressions should be enough.

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing (none for these pages)

### CODE REVIEW

File: media/js/foundation/annual2011.js [hunk 1]
```diff
@@ -18,71 +17,91 @@
...
-            }
-        },
-        function() {
-            if (wideMode) {
-                $(this).find(".overlay").delay(300).fadeOut(600, function(){
-                    $(this).removeAttr('style');
-                    $(this).prev('button.read').attr({'aria-pressed':"false"});
+    $('.overlay-wrap').hoverIntent(function() {
+        if (wideMode) {
+            var overlayheight = $(this).height() - 80;
```
Comment (schalkneethling) [inline media/js/foundation/annual2011.js]: Nit: `overlayHeight`

File: media/js/foundation/annual2011.js [hunk 2]
```diff
@@ -18,71 +17,91 @@
...
-                $(this).parents(".overlay-wrap").find(".overlay").css({
+
+            if ($(this).attr('aria-pressed') === 'false') {
+
+                $(this).attr({
+                    'aria-pressed': 'true',
+                    'aria-label': ARIAHIDE
+                });
+
+                var overlayheight = $(this).parents('.overlay-wrap').height() - 80;
```
Comment (schalkneethling) [inline media/js/foundation/annual2011.js]: Nit: Here is another `overlayHeight`

**Comment by schalkneethling** [comment]
Two nits, other than that r+ :sparkles:

**Comment by alexgibson** [comment]
@schalkneethling updated thanks

**Pull request merged**